src/dotnet/Chat/LinkPreview.cs

```diff
@@ -0,0 +1,25 @@
+using MemoryPack;
+using Stl.Fusion.Blazor;
+
+namespace ActualChat.Chat;
+
+[ParameterComparer(typeof(ByValueParameterComparer))]
+[DataContract, MemoryPackable(GenerateType.VersionTolerant)]
+public sealed partial record LinkPreview : IHasId<Symbol>, IRequirementTarget
```

Comment:
Needs IHasVersion<long>; ModifiedAt might be omitted, coz Version in our case is always ~ ModifiedAt (unless we want to distinguish between user-made and system-made modifications).